src/explore-education-statistics-common/src/components/SummaryListItem.tsx

```diff
@@ -36,7 +38,9 @@ const SummaryListItem = ({
         </dd>
       )}
       {actions && <dd className="govuk-summary-list__actions">{actions}</dd>}
-      {children && !actions && <dd className="govuk-summary-list__actions" />}
+      {children && !actions && showActions && (
```

Comment:
This might never be a combination in use... not sure though!

Reply:
removed